webcurator-ui/src/stores/permissions.ts

```diff
@@ -0,0 +1,45 @@
+import { ref } from 'vue';
+import { defineStore } from 'pinia';
+import { type UseFetchApis, useFetch } from '@/utils/rest.api';
+import type { Permission } from '@/types/permission';
+
+export const usePermissionStatusStore = defineStore('PermissionStatus', () => {
+  const loadingPermissionStatuses = ref(false);
+  const statuses = ref();
+  const rest: UseFetchApis = useFetch();
+
+  const fetch = async () => {
+    if (statuses.value) {       
```

Comment:
Shall we mandatory to fetch the statuses if the method is called? So the statuses can be refreshed from other operations.

Reply:
This just fetches the text for each status number, e.g. 1: "Scheduled", 2: "Queued" etc. So the values won't change.
I have actually moved it to the harvestAuthorisation store, since it uses the harvest-authorisation API, not permission.

Reply:
Cool.